lib/OpenQA/Worker/Jobs.pm

```diff
@@ -164,6 +175,21 @@ sub upload {
 
     my $res = $OpenQA::Worker::Common::ua->start($tx);
 
+    # Upload known server failures (Instead of anything that's not 200)
+    unless ($res->res->code =~ m/^5/) {
```

Comment:
@okurz In the end, it made more sense to remove the unless, when using is_status_class... ('cause using  `is_status_class(200)` doesn't make that much sense).